### PR TITLE
fix: badge link change

### PR DESCRIPTION
**Pull request by rachelcarmena**
**What**: Badge links to the Contributors section now.

**Why**: Reported in issue #195 

**How**:
1. Run tests :green_heart: 
2. Change tests to consider `#contributors-` as the link
3. Run tests :red_circle: 
4. Fix code to pass the tests: initial badge content and regular expressions.
5. Run tests :green_heart: 
6. I updated README.md as well.

**Checklist**:
- [ ] Documentation N/A
- [x] Tests 
- [x] Ready to be merged
- [x] Added myself to contributors table

Thanks for this opportunity to help with this awesome project!!

### CODE REVIEW

File: src/generate/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const _ = require('lodash/fp')
 const formatBadge = require('./format-badge')
 const formatContributor = require('./format-contributor')
 
-const badgeRegex = /\[!\[All Contributors\]\([a-zA-Z0-9\-./_:?=]+\)\]\(#\w+\)/
+const badgeRegex = /\[!\[All Contributors\]\([a-zA-Z0-9\-./_:?=]+\)\]\(#[\w\-]+\)/
```
Comment (Berkmann18) [inline src/generate/index.js]: > [lint] /home/circleci/repo/src/generate/index.js
> [lint]   5:76  error  Unnecessary escape character: \-  no-useless-escape

Comment (rachelcarmena) [reply src/generate/index.js]: Oh! I forgot to execute Lint... :woman_facepalming: 
Thanks @Berkmann18 , done!!

Comment (rachelcarmena) [reply src/generate/index.js]: @Berkmann18 , please, can I ask you a question? Why did Lint find only one of them? I mean, `-` is escaped before. It's only a curiosity. Thanks again for your help!!

Comment (rachelcarmena) [reply src/generate/index.js]: @Berkmann18 , it seems it only happens for `-`as the first element or the last element in a RE set. If it's the first or the last element, it's not indicating a range so it's not necessary. Thanks again!!

Comment (Berkmann18) [reply src/generate/index.js]: @rachelcarmena Normally the pre-commit hook should automatically action ESLint (depending on the file extension) so you either used `--no-verify` or it bugged somewhere.
> please, can I ask you a question? Why did EsLint find only one of them? I mean, - is escaped before. It's only a curiosity. Thanks again for your help!!

Good catch, I guess it only looked at the one changed but that wouldn't really make sense considering it's on the same line.

> it seems it only happens for -as the first element or the last element in a RE set. If it's the first or the last element, it's not indicating a range so it's not necessary. Thanks again!!

After reflecting on that, RE's grammar is such that `-` placed between characters (e.g: `[a-z]`, `[0-9]`) acts as the range operator (so /[a-z]/ would match any single lower case letters and /[0-9]/, any single digits). Whereas escaping it (`\-`) would tell the RE parser that it's an **element of the set**.

In other words: /[a-c]/ accepts 'a', 'b' and 'c' whereas /[a\-c]/ only accepts 'a', '-' and 'c'.

I hope that makes sense.

Comment (rachelcarmena) [reply src/generate/index.js]: Thanks @Berkmann18 ! It makes sense.
